syntax = "proto3";

service SSOVerifyUrl{
	rpc VerifyUrl(VerifyInRequest) returns (VerifyOutResponse){}
}

message VerifyInRequest{
	string RequestUrl = 1;
	string TicketId = 2;
}

message VerifyOutResponse{
	int32 Code = 1;
	string Message = 2;
	string Content = 3;
}
